ANNOUNCE.rst

```diff
@@ -43,6 +49,25 @@ bcolz has minimal dependencies (NumPy), comes with an exhaustive test
 suite and fully supports both 32-bit and 64-bit platforms.  Also, it is
 typically tested on both UNIX and Windows operating systems.
 
+Together, bcolz and the Blosc compressor, are finally fulfilling the
+promise of accelerating memory I/O, at least for some real scenarios:
+
+http://nbviewer.ipython.org/github/Blosc/movielens-bench/blob/master/querying-ep14.ipynb#Plots
+
+Other users of bcolz are the Blaze project (http://blaze.pydata.org/)
+and Quantopian (https://www.quantopian.com/) which you can read more
+about by pointing your browser at the links below.
+
+* Blaze links:
+
+  http://nbviewer.ipython.org/url/blaze.pydata.org/notebooks/timings-bcolz.ipynb
+  http://matthewrocklin.com/blog/work/2014/12/30/Towards-OOC-Frontend/
+  http://matthewrocklin.com/blog/work/2014/12/27/Towards-OOC/
```

Comment:
Sounds great! If you want, you can add a link to bquery for people lazy to google ;) https://github.com/visualfabriq/bquery